GitUI/UserControls/AvatarControl.Designer.cs

```diff
@@ -15,6 +15,7 @@ protected override void Dispose(bool disposing)
         {
             if (disposing && (components != null))
             {
+                _cancellationTokenSequence.Dispose();
```

Comment:
❗️ Isn't this a generated code file that will get generated again when the form is changed? If so, you should be able either use an event handler or create a wrapper `IComponent` that is added to `components` and delegates the `Dispose` implementation to this code.

Reply:
📝 If this code _doesn't_ change when controls are added to AvatarControl, let me know.

Reply:
I'd move it to the non designer file. See if designing the form adds the method back to designer.